### ISSUE TITLE
Subtitle with display formatting in card view and searching with more than 3 characters

### ISSUE THREAD

**Issue by terafirmanz** [utomic-media/directus-extension-field-actions#45]
In card view for a collection where you have values in the subtitle or title that use this extension for display formatting. when performing a search on the collection page, as you type the items will be filtered but once the search term goes over 3 characters the subtitle and or title disappears. This does not happen if using a table view.

For us we use this extension to format the subtitle on this collection to give a copy button for the ip address.
(sorry for having to blur but its production data)
items no search query 
![285303919-5ab49b9f-935c-426b-a58c-344f79ff2faf](https://github.com/utomic-media/directus-extension-field-actions/assets/53923271/3c783349-1e2f-475e-8db1-c8c472cd0fba)

Items search query 3 char:
![285304261-8111b5c8-fb23-42d8-980d-be6be4f64ada](https://github.com/utomic-media/directus-extension-field-actions/assets/53923271/908dc755-d8ff-4904-a770-bbb760811e26)

Items search query 4 char:
![285304539-b12ce8f1-8640-411a-b1bf-16b2ad5c7ed7](https://github.com/utomic-media/directus-extension-field-actions/assets/53923271/61d69d4c-1f9f-4ff3-8377-ad7ee4b60f28)

**Comment by terafirmanz** [comment]
Further to this I found if the click action is the display formatting is set to copy value rather than default action then the data is always not visible in the card view regardless of if a search is being performed or not.

**Comment by Dominic-Marcelino** [comment]
Hi, thanks for pointing out!
I haven't used the extension with the card layout yet. I'll make sure to test this next week and let you know if there's anything more I need for reproduction.

**Comment by Dominic-Marcelino** [comment]
Unfortunately I'm not able to replicate the issue.
Are you using the latest version?

**Search with display in "classic mode" with action-buttons next to the value:**
![Bildschirmfoto 2023-11-29 um 16 52 55](https://github.com/utomic-media/directus-extension-field-actions/assets/46492597/8f7a2b8a-8bcf-4479-89fd-a0337744a0da)

**Search with display in "button mode" with a custom action button:**
![Bildschirmfoto 2023-11-29 um 17 07 07](https://github.com/utomic-media/directus-extension-field-actions/assets/46492597/ed04c730-2b54-4c81-ad68-d12837eef42f)

Can you specify in which situations the issue happens (field-type, layout configuration, display config)?

---
_Apart from this, I notices that the button mode isn't fully compatible with the card-layout. The buttons are too high for the wrapper. Therefore I switched to a smaller button in a new version_

**Comment by terafirmanz** [comment]
Ok wrote a whole reply and just as I was about to post I tested a hunch and found the problem. its in the display layout subtitle having a space after a hyphen. ALso the problem only happens when the results filter down to a single row of results.

this has the issue:
![image](https://github.com/utomic-media/directus-extension-field-actions/assets/53923271/fd4cd6fc-f46a-46cb-a1d6-c1eb27014a1c)

thsi does not:
![image](https://github.com/utomic-media/directus-extension-field-actions/assets/53923271/73727149-28b6-44b1-8765-596e0825680f)

**Comment by Dominic-Marcelino** [comment]
Unfortunately I'm still not able to replicate the issue.

Next to my previous tests I tried to replicate your setup as much as possible from the given information:
- using a status field that's displayed as dots
- Adding a text-field with the display
- Tested all three default click options
- Tested different button modes

![Bildschirmfoto 2023-11-30 um 09 32 50](https://github.com/utomic-media/directus-extension-field-actions/assets/46492597/23a493f5-62ce-43ab-ac5c-8ba1ac81fe18)

Which versions of directus and this extension are you using? Have you installed the extension via npm or by copying the files into the extensions-folder?

**Comment by terafirmanz** [comment]
If you make the subtitle in the layout options:
Status <space> - <space> URL
you should get the problem.
I have noticed that updating to the latest version of your plugin has meant some records always show correctly even when others filter as well just to confuse me more.

**Comment by terafirmanz** [comment]
Will do. We are on 1 behind the latest 10.7 but will test more as other collections don’t do it to me.

**Comment by terafirmanz** [comment]
Ok so, I've updated everything to the latest directus 10.8.2 and this extension to 1.6.6 and still have the issue. I then tested on other collections and could re-create the problem. SO I tested on a new test collection and could not re-create the problem with random data but as soon as I started entering IP addresses in the field that was set to the subtitle display the problem happened.

It has to be exactly this setup in the card subtitle display for the problem to happen:
{status}<space>-<space>{field with IP data}
if the data is not IPv4 data (numbers with dots) the problem does not happen. So I don't know if this is a directus css issue or this extension but it only happens when this extension is active in the display of the affected field.
